session/src/main/scala/akka/cassandra/session/DriverConfigLoaderFromConfig.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
wow, is that the settings of the header plugin?

Reply:
Old school